.github/workflows/release.yaml

```
@@ -274,10 +274,7 @@ jobs:
     uses: open-component-model/.github/.github/workflows/bdba.yaml@main
     with:
       artifact_id: ctf-aggregated
-    secrets:
-      BDBA_API_TOKEN: ${{ secrets.BDBA_API_TOKEN }}
-      BDBA_URL: ${{ secrets.BDBA_URL }}
-      BDBA_GROUP_ID: ${{ secrets.BDBA_GROUP_ID }}
+    secrets: inherit
```

```suggestion
      BDBA_API_TOKEN: ${{ secrets.BDBA_API_TOKEN }}
      BDBA_URL: ${{ secrets.BDBA_URL }}
      BDBA_GROUP_ID: ${{ secrets.BDBA_GROUP_ID }}
      BDBA_USERNAME: ${{ secrets.BDBA_USERNAME }}
      BDBA_PASSWORD: ${{ secrets.BDBA_PASSWORD }}
```

The required fields were updated/extended in the reusable workflow, but are not present on this release branch

---

Inheriting the secrets for us in this is acceptable

---

as the names are the same, this should work